### DISCUSSION TITLE
Vertical stepper

### DISCUSSION THREAD

**Discussion by ThcDelux3** [vueform/vueform#79] [Ideas]
### Describe the feature

Hey a vertical stepper option would be awesome.

Like here: https://m1.material.io/components/steppers.html

### Additional information

- [X] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a plugin?

### Final checks

- [X] Read the [contribution guide](https://vueform.com/community/contribution-guide).
- [X] Check existing [discussions](https://github.com/vueform/vueform/discussions) and [ideas](https://github.com/vueform/vueform/issues/categories/ideas).
- [X] Check [roadmap](https://github.com/vueform/builder/discussions/categories/roadmap).

**Comment by adamberecz** [comment]
I love the idea! Would you need any input from me? Maybe you want to share how would you approach this?

**Comment by saenzramiro** [comment]
I'm very interested in having the steps vertically. Any workaround for this?
